app/services/discover/register.rb

```diff
@@ -0,0 +1,34 @@
+module Discover
+  class Register
+    FOREM_DISCOVER_URL = "https://discover.forem.com/api/forems/register".freeze
+
+    def initialize(domain:)
```

Comment:
Could it make sense to specify a default here? It seems like `Settings::General.app_domain` would generally be what we want.

```suggestion
    def initialize(domain: Settings::General.app_domain)
```